routes/contract.js

```diff
@@ -32,12 +56,51 @@ router.post("/create", async (req, res) => {
   }
 });
 
+/** /contract/details/:contractNo */
+router.get("/details/:contractNo", async (req, res) => {
+  try {
+    let contractNo = req.params.contractNo;
+    let contractContent = await getContractContentByContractNo(contractNo);
+    return res.status(201).json({ status: 'success', message: 'Get contract content successfully', contractContent });
+  } catch (err) {
+    console.error(`Error while getting contract content`);
+    console.error(err);
+    return res.status(500).json({ status : 'error', message : err.message });
+  }
+});
+
+/** /contract/details/:contractNo */
+router.post("/details/:contractNo", async (req, res) => {
+  try {
+    let contractNo = req.params.contractNo;
+    let secondParty = req.body.secondPartySignature;
+    await updateSecondPartySignature(contractNo, secondParty);
+    let updatedContent = await updateSecondParty(contractNo, secondParty);
```

Comment:
since u don't have dependency on these two awaits you can call them parallel which would speed your API
It would look something like this await Promise.all([updateSecondPartySignature, updateScondParty])
This will make two requests at a time and say suppose each request takes 1second to execute, Promise.all will take only 1 second to execute both of them, where as two awaits will make blocking and will take 1second + 1 second = 2 seconds